config/dispatcher/dispatcher.ex

```diff
@@ -75,6 +75,42 @@ defmodule Dispatcher do
     Proxy.forward conn, path, "http://frontend/@appuniversum/"
   end
 
+  match "/bestuurseenheid-classificatie-codes/*path", %{layer: :api, accept: %{json: true}} do
```

Comment:
most projects I worked on it said resource layer but we work with api layer right?